app/views/courses/preview.html.erb

```diff
@@ -0,0 +1,94 @@
+<%= content_for :page_title, "#{course.name_and_code} - Courses" %>
+
+<% content_for :before_content do %>
+  <%= link_to "Back to course", provider_course_path, class: "govuk-back-link" %>
+
+  <p class="govuk-body preview-banner">
+    This is a preview of a course: <%= course.name_and_code %>.
+  </p>
+<% end %>
+
+<h1 class="govuk-heading-xl">
+  <span class="govuk-!-font-size-36"><%= course.provider.provider_name %></span><br>
+  <%= course.name_and_code %>
+</h1>
+
+<p class="govuk-body-l"><%= course.description %></p>
+
+<dl class="govuk-list--description">
+  <dt class="govuk-list--description__label">Qualification</dt>
+  <dd>
+    <%= render partial: 'courses/preview/qualification' %>
+  </dd>
+  <% if course.length.present? %>
+    <dt class="govuk-list--description__label">Course length</dt>
+    <dd><%= course.length %></dd>
+  <% end %>
+  <% if course.applications_open_from.present? %>
+    <dt class="govuk-list--description__label">Date you can apply from</dt>
+    <dd><%= l(course.applications_open_from&.to_date) %></dd>
+  <% end %>
+  <% if course.applications_open_from.present? %>
```

Comment:
Should this and the section above be separate? They're testing the same thing ...